tests/functional/modules/test_zos_job_submit_func.py

```diff
@@ -398,8 +401,9 @@
 //
 """
 
-TEMP_PATH = "/tmp/jcl"
-DATA_SET_NAME_SPECIAL_CHARS = "imstestl.im@1.x#$xx05"
+def get_unique_uss_file_name():
```

Comment:
So no helper function in one of the setup scripts we have? This would be amazing to have available to the whole collection, I noticed you have this function in zos_fetch too